### DISCUSSION TITLE
How can I return to my editing file without closing neo-tree?

### DISCUSSION THREAD

**Discussion by KentVu** [nvim-neo-tree/neo-tree.nvim#1516] [Q&A]
My action's kinda like pressing Esc in Intellij IDEA, if you press that anywhere outside the editing document, it'll return you to the editing document.

In nvim that would be equivalent to press `<C-W><C-W>`, but I want to remap it to <kbd>Esc</kbd>, is that possible inside neo-tree?

**Comment by miversen33** [comment]
Neo-tree is just a window within Neovim. I suppose you could remap the `Esc` key within Neo-tree to be `<C-W><C-W>` (see [`:h neo-tree-custom-mappings`](https://github.com/nvim-neo-tree/neo-tree.nvim/blob/main/doc/neo-tree.txt#L437) for details) but you should really adjust your framing here. Neo-tree is just a window in your editor. So treat it as one. Close it when you are done, or dont and leave it open by navigating away to another window (as you mentioned with the keycommands to move between windows).

**Reply by KentVu** [reply]
Thank you, I've resorted to this:
```lua
     window = {
        mappings = {
          ["<Esc>"] = function(state)
            vim.cmd("wincmd l")
          end,
...
```
